app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
```

Comment:
- it's recommended to use single style for imports in all files of one program
- it's better to use ( ) while importing multiple classes/modules